msktutil.h

```diff
@@ -118,6 +118,18 @@
 #define MS_KERB_ENCTYPE_AES256_CTS_HMAC_SHA1_96 0
 #endif
 
+#define DEFAULT_MS_KERB_ENCTYPES \
```

Comment:
In order not to have surprises with macro expansion we should have braces () around the expression

Reply:
You mean at all places used?

Reply:
Yes, expressions within macros should have (unless intentionally) braces.

Reply:
Will do!

Reply:
Done